test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -528,6 +504,81 @@ def setUp(self):
         self.log_end_setup()
         self.measurements = []
 
+    @classmethod
+    def populate_test_list(cls):
+        cls.test_list = [attr for attr in dir(cls)
```

Comment:
I think we should make this dict of (name, obj) (aka (attr, getattr(cls, attr)) so in below func we don't have to call getattr again for each test